test/testing-helpers.ts

```diff
@@ -26,6 +26,48 @@ import { TemplateResult } from '@spectrum-web-components/base';
 
 import { sendMouse } from './plugins/browser.js';
 
+export async function sendMouseTo(
```

Comment:
I found the tests hard to read with all the `SendMouse` code that basically does the same thing, so I added this